ERROR - 2024-05-25 10:45:20 --> Error connecting to the database: mysqli_sql_exception: Network is unreachable in /Users/unaibueno/Desktop/Mond/system/Database/MySQLi/Connection.php:186
Stack trace:
#0 /Users/unaibueno/Desktop/Mond/system/Database/MySQLi/Connection.php(186): mysqli->real_connect('212.227.231.241', 'unaibueno', Object(SensitiveParameterValue), 'mond_db', 3306, '', 0)
#1 /Users/unaibueno/Desktop/Mond/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /Users/unaibueno/Desktop/Mond/system/Database/BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 /Users/unaibueno/Desktop/Mond/system/Database/BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /Users/unaibueno/Desktop/Mond/system/Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#5 /Users/unaibueno/Desktop/Mond/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /Users/unaibueno/Desktop/Mond/app/Controllers/CalendarioController.php(53): CodeIgniter\BaseModel->findAll()
#7 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(933): App\Controllers\CalendarioController->getEvents()
#8 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CalendarioController))
#9 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /Users/unaibueno/Desktop/Mond/system/Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 /Users/unaibueno/Desktop/Mond/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 /Users/unaibueno/Desktop/Mond/public/index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 /Users/unaibueno/Desktop/Mond/system/Commands/Server/rewrite.php(49): require_once('/Users/unaibuen...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Network is unreachable in /Users/unaibueno/Desktop/Mond/system/Database/MySQLi/Connection.php:231
Stack trace:
#0 /Users/unaibueno/Desktop/Mond/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /Users/unaibueno/Desktop/Mond/system/Database/BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 /Users/unaibueno/Desktop/Mond/system/Database/BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /Users/unaibueno/Desktop/Mond/system/Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#4 /Users/unaibueno/Desktop/Mond/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 /Users/unaibueno/Desktop/Mond/app/Controllers/CalendarioController.php(53): CodeIgniter\BaseModel->findAll()
#6 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(933): App\Controllers\CalendarioController->getEvents()
#7 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CalendarioController))
#8 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /Users/unaibueno/Desktop/Mond/system/Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 /Users/unaibueno/Desktop/Mond/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /Users/unaibueno/Desktop/Mond/public/index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 /Users/unaibueno/Desktop/Mond/system/Commands/Server/rewrite.php(49): require_once('/Users/unaibuen...')
#13 {main}
CRITICAL - 2024-05-25 10:45:20 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Network is unreachable
[Method: GET, Route: get-events]
in SYSTEMPATH/Database/BaseConnection.php on line 457.
 1 SYSTEMPATH/Database/BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `calendario`', [], false)
 3 SYSTEMPATH/Model.php(275): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH/Controllers/CalendarioController.php(53): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\CalendarioController->getEvents()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CalendarioController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/Commands/Server/rewrite.php(49): require_once('/Users/unaibueno/Desktop/Mond/public/index.php')
ERROR - 2024-05-25 10:45:23 --> Error connecting to the database: mysqli_sql_exception: Network is unreachable in /Users/unaibueno/Desktop/Mond/system/Database/MySQLi/Connection.php:186
Stack trace:
#0 /Users/unaibueno/Desktop/Mond/system/Database/MySQLi/Connection.php(186): mysqli->real_connect('212.227.231.241', 'unaibueno', Object(SensitiveParameterValue), 'mond_db', 3306, '', 0)
#1 /Users/unaibueno/Desktop/Mond/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /Users/unaibueno/Desktop/Mond/system/Database/BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 /Users/unaibueno/Desktop/Mond/system/Database/BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /Users/unaibueno/Desktop/Mond/system/Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#5 /Users/unaibueno/Desktop/Mond/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /Users/unaibueno/Desktop/Mond/app/Controllers/CalendarioController.php(53): CodeIgniter\BaseModel->findAll()
#7 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(933): App\Controllers\CalendarioController->getEvents()
#8 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CalendarioController))
#9 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /Users/unaibueno/Desktop/Mond/system/Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 /Users/unaibueno/Desktop/Mond/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 /Users/unaibueno/Desktop/Mond/public/index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 /Users/unaibueno/Desktop/Mond/system/Commands/Server/rewrite.php(49): require_once('/Users/unaibuen...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Network is unreachable in /Users/unaibueno/Desktop/Mond/system/Database/MySQLi/Connection.php:231
Stack trace:
#0 /Users/unaibueno/Desktop/Mond/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /Users/unaibueno/Desktop/Mond/system/Database/BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 /Users/unaibueno/Desktop/Mond/system/Database/BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /Users/unaibueno/Desktop/Mond/system/Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#4 /Users/unaibueno/Desktop/Mond/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 /Users/unaibueno/Desktop/Mond/app/Controllers/CalendarioController.php(53): CodeIgniter\BaseModel->findAll()
#6 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(933): App\Controllers\CalendarioController->getEvents()
#7 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CalendarioController))
#8 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /Users/unaibueno/Desktop/Mond/system/Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 /Users/unaibueno/Desktop/Mond/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /Users/unaibueno/Desktop/Mond/public/index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 /Users/unaibueno/Desktop/Mond/system/Commands/Server/rewrite.php(49): require_once('/Users/unaibuen...')
#13 {main}
CRITICAL - 2024-05-25 10:45:23 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Network is unreachable
[Method: GET, Route: get-events]
in SYSTEMPATH/Database/BaseConnection.php on line 457.
 1 SYSTEMPATH/Database/BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `calendario`', [], false)
 3 SYSTEMPATH/Model.php(275): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH/Controllers/CalendarioController.php(53): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\CalendarioController->getEvents()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CalendarioController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/Commands/Server/rewrite.php(49): require_once('/Users/unaibueno/Desktop/Mond/public/index.php')
ERROR - 2024-05-25 10:45:49 --> Error connecting to the database: mysqli_sql_exception: Network is unreachable in /Users/unaibueno/Desktop/Mond/system/Database/MySQLi/Connection.php:186
Stack trace:
#0 /Users/unaibueno/Desktop/Mond/system/Database/MySQLi/Connection.php(186): mysqli->real_connect('212.227.231.241', 'unaibueno', Object(SensitiveParameterValue), 'mond_db', 3306, '', 0)
#1 /Users/unaibueno/Desktop/Mond/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /Users/unaibueno/Desktop/Mond/system/Database/BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 /Users/unaibueno/Desktop/Mond/system/Database/BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /Users/unaibueno/Desktop/Mond/system/Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#5 /Users/unaibueno/Desktop/Mond/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /Users/unaibueno/Desktop/Mond/app/Controllers/CalendarioController.php(53): CodeIgniter\BaseModel->findAll()
#7 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(933): App\Controllers\CalendarioController->getEvents()
#8 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CalendarioController))
#9 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /Users/unaibueno/Desktop/Mond/system/Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 /Users/unaibueno/Desktop/Mond/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 /Users/unaibueno/Desktop/Mond/public/index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 /Users/unaibueno/Desktop/Mond/system/Commands/Server/rewrite.php(49): require_once('/Users/unaibuen...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Network is unreachable in /Users/unaibueno/Desktop/Mond/system/Database/MySQLi/Connection.php:231
Stack trace:
#0 /Users/unaibueno/Desktop/Mond/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /Users/unaibueno/Desktop/Mond/system/Database/BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 /Users/unaibueno/Desktop/Mond/system/Database/BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /Users/unaibueno/Desktop/Mond/system/Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#4 /Users/unaibueno/Desktop/Mond/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 /Users/unaibueno/Desktop/Mond/app/Controllers/CalendarioController.php(53): CodeIgniter\BaseModel->findAll()
#6 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(933): App\Controllers\CalendarioController->getEvents()
#7 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CalendarioController))
#8 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /Users/unaibueno/Desktop/Mond/system/Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 /Users/unaibueno/Desktop/Mond/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /Users/unaibueno/Desktop/Mond/public/index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 /Users/unaibueno/Desktop/Mond/system/Commands/Server/rewrite.php(49): require_once('/Users/unaibuen...')
#13 {main}
CRITICAL - 2024-05-25 10:45:49 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Network is unreachable
[Method: GET, Route: get-events]
in SYSTEMPATH/Database/BaseConnection.php on line 457.
 1 SYSTEMPATH/Database/BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `calendario`', [], false)
 3 SYSTEMPATH/Model.php(275): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH/Controllers/CalendarioController.php(53): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\CalendarioController->getEvents()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CalendarioController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/Commands/Server/rewrite.php(49): require_once('/Users/unaibueno/Desktop/Mond/public/index.php')
ERROR - 2024-05-25 10:45:59 --> Error connecting to the database: mysqli_sql_exception: Network is unreachable in /Users/unaibueno/Desktop/Mond/system/Database/MySQLi/Connection.php:186
Stack trace:
#0 /Users/unaibueno/Desktop/Mond/system/Database/MySQLi/Connection.php(186): mysqli->real_connect('212.227.231.241', 'unaibueno', Object(SensitiveParameterValue), 'mond_db', 3306, '', 0)
#1 /Users/unaibueno/Desktop/Mond/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /Users/unaibueno/Desktop/Mond/system/Database/BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 /Users/unaibueno/Desktop/Mond/system/Database/BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /Users/unaibueno/Desktop/Mond/system/Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#5 /Users/unaibueno/Desktop/Mond/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /Users/unaibueno/Desktop/Mond/app/Controllers/CalendarioController.php(53): CodeIgniter\BaseModel->findAll()
#7 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(933): App\Controllers\CalendarioController->getEvents()
#8 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CalendarioController))
#9 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /Users/unaibueno/Desktop/Mond/system/Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 /Users/unaibueno/Desktop/Mond/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 /Users/unaibueno/Desktop/Mond/public/index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 /Users/unaibueno/Desktop/Mond/system/Commands/Server/rewrite.php(49): require_once('/Users/unaibuen...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Network is unreachable in /Users/unaibueno/Desktop/Mond/system/Database/MySQLi/Connection.php:231
Stack trace:
#0 /Users/unaibueno/Desktop/Mond/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /Users/unaibueno/Desktop/Mond/system/Database/BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 /Users/unaibueno/Desktop/Mond/system/Database/BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /Users/unaibueno/Desktop/Mond/system/Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#4 /Users/unaibueno/Desktop/Mond/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 /Users/unaibueno/Desktop/Mond/app/Controllers/CalendarioController.php(53): CodeIgniter\BaseModel->findAll()
#6 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(933): App\Controllers\CalendarioController->getEvents()
#7 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CalendarioController))
#8 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /Users/unaibueno/Desktop/Mond/system/Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 /Users/unaibueno/Desktop/Mond/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /Users/unaibueno/Desktop/Mond/public/index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 /Users/unaibueno/Desktop/Mond/system/Commands/Server/rewrite.php(49): require_once('/Users/unaibuen...')
#13 {main}
CRITICAL - 2024-05-25 10:45:59 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Network is unreachable
[Method: GET, Route: get-events]
in SYSTEMPATH/Database/BaseConnection.php on line 457.
 1 SYSTEMPATH/Database/BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `calendario`', [], false)
 3 SYSTEMPATH/Model.php(275): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH/Controllers/CalendarioController.php(53): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\CalendarioController->getEvents()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CalendarioController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/Commands/Server/rewrite.php(49): require_once('/Users/unaibueno/Desktop/Mond/public/index.php')
ERROR - 2024-05-25 10:46:01 --> Error connecting to the database: mysqli_sql_exception: Network is unreachable in /Users/unaibueno/Desktop/Mond/system/Database/MySQLi/Connection.php:186
Stack trace:
#0 /Users/unaibueno/Desktop/Mond/system/Database/MySQLi/Connection.php(186): mysqli->real_connect('212.227.231.241', 'unaibueno', Object(SensitiveParameterValue), 'mond_db', 3306, '', 0)
#1 /Users/unaibueno/Desktop/Mond/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /Users/unaibueno/Desktop/Mond/system/Database/BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 /Users/unaibueno/Desktop/Mond/system/Database/BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /Users/unaibueno/Desktop/Mond/system/Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#5 /Users/unaibueno/Desktop/Mond/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 /Users/unaibueno/Desktop/Mond/app/Controllers/CalendarioController.php(53): CodeIgniter\BaseModel->findAll()
#7 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(933): App\Controllers\CalendarioController->getEvents()
#8 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CalendarioController))
#9 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /Users/unaibueno/Desktop/Mond/system/Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 /Users/unaibueno/Desktop/Mond/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 /Users/unaibueno/Desktop/Mond/public/index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 /Users/unaibueno/Desktop/Mond/system/Commands/Server/rewrite.php(49): require_once('/Users/unaibuen...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Network is unreachable in /Users/unaibueno/Desktop/Mond/system/Database/MySQLi/Connection.php:231
Stack trace:
#0 /Users/unaibueno/Desktop/Mond/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /Users/unaibueno/Desktop/Mond/system/Database/BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 /Users/unaibueno/Desktop/Mond/system/Database/BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /Users/unaibueno/Desktop/Mond/system/Model.php(275): CodeIgniter\Database\BaseBuilder->get()
#4 /Users/unaibueno/Desktop/Mond/system/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 /Users/unaibueno/Desktop/Mond/app/Controllers/CalendarioController.php(53): CodeIgniter\BaseModel->findAll()
#6 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(933): App\Controllers\CalendarioController->getEvents()
#7 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CalendarioController))
#8 /Users/unaibueno/Desktop/Mond/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /Users/unaibueno/Desktop/Mond/system/Boot.php(312): CodeIgniter\CodeIgniter->run()
#10 /Users/unaibueno/Desktop/Mond/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /Users/unaibueno/Desktop/Mond/public/index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 /Users/unaibueno/Desktop/Mond/system/Commands/Server/rewrite.php(49): require_once('/Users/unaibuen...')
#13 {main}
CRITICAL - 2024-05-25 10:46:01 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Network is unreachable
[Method: GET, Route: get-events]
in SYSTEMPATH/Database/BaseConnection.php on line 457.
 1 SYSTEMPATH/Database/BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `calendario`', [], false)
 3 SYSTEMPATH/Model.php(275): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH/Controllers/CalendarioController.php(53): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\CalendarioController->getEvents()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CalendarioController))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(57): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH/Commands/Server/rewrite.php(49): require_once('/Users/unaibueno/Desktop/Mond/public/index.php')
